pcdsdevices/pim.py

```diff
@@ -372,24 +372,38 @@ class PPMPowerMeter(BaseInterface, Device):
     calibrated for each power meter in units of volts per watt
     is used to calculate the actual energy in the following way:
 
-    `calibrated_mj` = 1000 * (Signal - Background) / (Responsivity * Beam_Rate)
+    `calibrated_mj` = (Signal - Background) / (Responsivity * Beam_Rate)
     """
+    def __init__(self, prefix, *, name, **kwargs):
+        super().__init__(prefix, name=name, **kwargs)
```

Comment:
Just realizing this is here, we don't really need this init do we?

Reply:
No